.circleci/check-testnet-deployment.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+const { waitUntilSync, getBlockHeight, getCommit } = require('@orbs-network/orbs-nebula/lib/metrics');
+const fetch = require('node-fetch');
+
+const TOPOLOGY = 'https://s3.us-west-2.amazonaws.com/boyar-testnet-bootstrap/boyar/config.json';
+const TARGET_HASH = process.argv[2];
+
+if (!TARGET_HASH) {
+    console.log('No target hash to check');
+    process.exit(1);
+}
+
+async function getTopology() {
+    const result = await fetch(TOPOLOGY);
+    const topology = await result.json();
+    return topology;
+}
+
+async function eventuallyDeployed({ chainId, nodes }) {
```

Comment:
`waitUnitlCommit` from `metrics` does that already: https://github.com/orbs-network/nebula/blob/master/lib/metrics.js#L112

Reply:
I do not agree. I want to see all nodes have updated to the new version. This is not the same.
Your method is single endpoint centric

Reply:
You can run it multiple times waiting for the same commit. Same effect, less code.